docs/UserGuide.md

```diff
@@ -204,6 +204,21 @@ Examples:
 * `filter vendor customer` returns all contacts tagged by the tag: vendor and
   all contacts tagged by the tag: customer in JobFestGo.
 
+
+### Adding an event: `add_event`
+
+Adds an event to JobFestGo.
+
+Format: `add_event n/NAME d/DATE a/ADDRESS`
+
+* An event must have mandatory fields name, date and address
+* The event name **must not already exist** in JobFestGo.
+* Date must be valid should be in the appropriate (YYYY-MM-DD) format
```

Comment:
perhaps `Date should be in the appropriate format: YYYY-MM-DD` is enough or a proper sentence: `Date must be valid and should be in the appropriate (YYYY-MM-DD) format`

Reply:
fixed